server/controllers/middlewares.js

```diff
@@ -257,4 +258,17 @@ export const sorting = (options) => {
 
     next();
   };
+};
+
+/**
+ * Populate req with info on whether this user can edit colletive
+ */
+export const populateEditPermissions = (req, res, next) => {
+  if (req.remoteUser && req.group) {
+    return req.group.hasUserWithRole(req.remoteUser.id, [roles.HOST, roles.MEMBER])
+    .then(canEditGroup => req.remoteUser.canEditGroup = canEditGroup)
```

Comment:
I wonder if it's not confusing to have `req.remoteUser.canEditGroup` because it's not very explicit on which group this is referring to. What about having instead `req.remoteUser.canEditGroup(groupId)` or `req.remoteUser.canEditGroup[groupId]` to avoid any confusion.